transformations/mr_value_replacement/transformation.py

```diff
@@ -0,0 +1,88 @@
+import torchtext.vocab
+import torch
+from torchtext.data import get_tokenizer
+from interfaces.KeyValuePairsOperation import KeyValuePairsOperation
+import spacy
+from itertools import product
+import random
+from random import sample
+from tasks.TaskTypes import TaskType
+
+class MRValueReplacement(KeyValuePairsOperation):
+    tasks = [TaskType.E2E_TASK]
+    languages = ["en"]
+  
+    def __init__(
```

Comment:
It would be great if you could add [PEP 257](https://www.python.org/dev/peps/pep-0257/) style comments to all the functions.

Reply:
Many thanks for your suggestions. I'll do the changes

Reply:
Done!

Reply:
Hi @msobrevillac, the DocStrings for each function is incomplete. You need to add DocStrings for each parameter and return vale for each function.